src/Points/points_fastapi.py

```
@@ -0,0 +1,175 @@
+                "success": False,
+                "message": "Failed to save point."
+            }
+        )
+
+# route which deletes a saved point that is passed into the back-end from the front-end
+@router.post(
+    "/points/delete-point",
+    dependencies=[Depends(RateLimiter(limiter=Limiter(Rate(110, Duration.MINUTE * 1))))])
+def delete_point(
+    point: PointSchema,
+    user: User | None = Depends(get_current_user)
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)